OpentokReactNative.podspec

```diff
@@ -19,9 +19,7 @@ Pod::Spec.new do |s|
   s.public_header_files = "ios/**/*.h" 
   
   # Add OTXCFramework dependency
-  s.dependency 'OTXCFramework', '2.30.1'
-  # Add VonageClientSDKVideoTransformers dependency
-  s.dependency 'VonageClientSDKVideoTransformers', '2.30.1'
```

Comment:
The SDK should not load this. The developer should add it to their project if they are using video transformers (as noted in the docs).